### PR TITLE
show result name

### LINKED ISSUE #179: Deleting existing result name hides field
Repro:
* Open node with a result name
* Focus at the end of the name
* Delete characters one by one to give a new name (or select all and delete)
* Result Name field vanishes

Expected:
Result Name field should never vanish while the user has focus on it

### PR DESCRIPTION
**Pull request by ycleptkellan**
fixes #179

### CODE REVIEW

**Comment by ericnewcomer** [comment]
How hard is it to add a test for this?

**Comment by ycleptkellan** [comment]
not so hard, happy to add

**Comment by ycleptkellan** [comment]
ok added a few tests for result name display

File: src/component/NodeEditor/NodeEditor.test.tsx [hunk 1]
```diff
@@ -221,6 +224,48 @@ describe('NodeEditor >', () => {
...
+                />
+            );
+
+            expect(getSpecWrapper(wrapper, 'name-field').prop('value')).toBe(
+                waitNode.router.result_name
+            );
+
+            wrapper.setState({ resultName: '' });
+
+            expect(getSpecWrapper(wrapper, 'name-field').prop('value')).toBe('');
```
Comment (ericnewcomer) [inline src/component/NodeEditor/NodeEditor.test.tsx]: is this ensuring that it is visible?

Comment (ycleptkellan) [reply src/component/NodeEditor/NodeEditor.test.tsx]: ya the result of the `getSpecWrapper` call is the `TextInputElement` so if this doesn't throw it's both visible and it contains an empty string

Comment (ericnewcomer) [reply src/component/NodeEditor/NodeEditor.test.tsx]: Ok, that’s helpful. Did you confirm the test failure without your fix?

**Comment by ycleptkellan** [comment]
@ericnewcomer updated test to be clearer, fail for previous case

**Pull request merged**